### ISSUE TITLE
Current Status.

### ISSUE THREAD

**Issue by filfat** [filfat/swiftsnapper#2]
This is a quick issue where we can report status updates, such as screenshots etc.

**Comment by filfat** [comment]
![img](http://i.imgur.com/IdCZGSc.png)
*Account*

![img](http://i.imgur.com/TCM7TU5.png)
*Dashboard*

**Comment by KyleMcNutt** [comment]
It's probably not a good idea to have this as an issue on the repo, but is probably handled much better in a [GitHub Wiki](https://help.github.com/articles/about-github-wikis/). This is your repo though so feel free to handle it however you would like, I just thought I would give my thoughts about what might make more sense for developers who are new to this project.

**Comment by Thomasfda** [comment]
Hello i'm french sorry for my english.. ! 

How long will the development take ?

**Comment by allan991** [comment]
So if a Snapchat see this. They could report to microsoft and banish from the windows store?

**Comment by Thomasfda** [comment]
I  think that the application does not go to the store, there will be an update out blind system ? It will install the application manually I think, as the android apk

**Comment by ChrisProlls** [comment]
It's the best solution I think. No store, no problem :)

**Comment by filfat** [comment]
@KyleMcNutt This is more a place to post screenshots etc so non-developers can keep up :)

Speaking of which
![](http://i.imgur.com/xoU8AvK.png)
We're now simulating on call kind-of corrctly; a long way to go ;)

**Comment by Cenite** [comment]
The logo kinda reminds me of Cortana. Is it still WIP or is the final one?

**Comment by filfat** [comment]
@Cenite not decided yet :)
I kind of like it though.

**Comment by filfat** [comment]
![Logged in!!!](http://imgur.com/qdPSXZS.png)
Soooo... I successfully logged in to my testing account! :grinning:

**Comment by Thomasfda** [comment]
Oh yeah it's good :D !!  i wait a beta of application..

**Comment by kikothiago** [comment]
Please continue with this APP snapchat for windows. We are very grateful.

**Comment by allan991** [comment]
Nice! keep testing I hope snapchat can't "see" we log in in to the service.

**Comment by jimmyjango** [comment]
Awesome. When do you expect people to begin to start testing / using it?

**Comment by filfat** [comment]
@allan991 During my hour of logging in over 50 times it's safe to say that so far we don't do anything differently from the first-party client.

@jimmyjango Hard to say, but a lot sooner than I initially thought :laughing: 
If you have experience with the Cordova CLI so can you already test c498e57e0453c88da6efed999ee0f2fa43a50854 with the possibility to login and view a list of all the unread snaps from the first friend (not recommend yet as there might still be an risk for account banning).

Cheers!

**Comment by gncgabriel** [comment]
When the Astoria Project allowed to install Android apps on Windows Phone, I tested the casper in my Windows phone and the app was required to log in using a google account and then the Snapchat account, in this app that you are developing, is also like this?

**Comment by filfat** [comment]
@gncgabriel I thought that would be an requirement initially, so far I have yet to run into any road blocks that require me to also auth a disposable google account :)

**Comment by jimmyjango** [comment]
@filfat Exciting. Thanks for keeping this thread up to date with status. Really helps keep updated on progress.

**Comment by utopy** [comment]
hey! sadly i'm not that good with cordova so i can't help you with that, but if u need a tester to get feedback! Contact me, it'll be my pleasure to help you bring a nice snapchat experience on windows phone!

**Comment by vitorgrs** [comment]
I want to sugest somethings. It will be better if you use the default colors of snapchat. The green is too ugly :yum:

**Comment by Vandorann** [comment]
If you are in need of people to test Swiftsnapper, I'll gladly help out however I can. Keep up the amazing work, @filfat!

**Comment by sartero** [comment]
It seems to me already won some few to help test the project, I think this a good idea because it will most likely somebody find a bug or something. I am aq to give my support, seprecisar somebody to test tell me too.
Project is getting great.

**Comment by Cenite** [comment]
Same here, if you need a tester you can count with me!😉

**Comment by allan991** [comment]
Yeah! when the app will be stable then let us use the app for several test with account login ( I have 3 account test on snapchat) and I will be glad to log I'm a hundred times and report bugs and have some ideas to UI n UX app experience

**Comment by filfat** [comment]
Can as many people as possible add one of my testing accounts "ceoatcompany" and send me a bunch of snaps once i've added you back? I need to figure out exactly how the unread array works :)

**Comment by filfat** [comment]
Apparently there's a subreddit for those interested https://www.reddit.com/r/swiftsnapper/ :)

**Comment by gncgabriel** [comment]
I have a doubt, I can Login directly in the app or do I have to change something in the project?

**Comment by filfat** [comment]
@gncgabriel No, you should be able to login using the UI; not that you can really do anything yet :)

**Comment by gncgabriel** [comment]
I understand very little of it, I have Visual Studio on my computer and now I was able to test this project in my Windows Phone

**Comment by JDHD98** [comment]
Is there going to be the same filters as snapchat and are there going to be face tracking features?

**Comment by filfat** [comment]
@JDHD98 Filters? Yes in the future.
face tracking feature? Ha ha, no... not by me in that case; I have no experience with face recognition :)

**Comment by hypertime123** [comment]
@filfat will the filters all be free, cause on snapchat most of them cost money. (Unless u Jb ur phone like me😂)

**Comment by filfat** [comment]
@hypertime123 Of course, this is an open source project with the intent of staying free and ad free. Though I might ask for donations in the future to rent servers in the cloud for something like #7 and testing devices :)

**Comment by hypertime123** [comment]
@filfat a little off topic but Would u ever have time to make a kik client cause I think the current one for windows is a little out of date. They are kind of in the same situation as snapchat, they stopped development for the app on windows phone after first version in 2013.

**Comment by filfat** [comment]
@hypertime123 One thing at a time :)

**Comment by hypertime123** [comment]
@filfat ok but would u ever consider it, thx

**Comment by pvrs12** [comment]
Ah you, found the subreddit I quickly hobbled together. Nothing special there yet, but if you have an account @filfat I'll give you mod privileges. Heck if you want I'll give you the whole subreddit

**Comment by filfat** [comment]
@pvrs12 Feel free to add "@filfat" :)

**Comment by gncgabriel** [comment]
eu consegui logar (eu acho), e a camera em smartphones está 90° á direita

**Comment by gncgabriel** [comment]
I could log in (I think), and the camera on smartphones is 90 ° to the right

**Comment by hypertime123** [comment]
@filfat do notifications work yet

**Comment by Thomasfda** [comment]
My visual studio "Project imcompatible" wait and see :(..

**Comment by gncgabriel** [comment]
@Thomasfda you have to install Apache Cordova in your Visual Studio

**Comment by Thomasfda** [comment]
Oooh shit.. ! Go install @gncgabriel Thanks !

**Comment by Thomasfda** [comment]
It is impossible to connect again?

**Comment by filfat** [comment]
@Thomasfda nope, works for me

**Comment by Thomasfda** [comment]
Ok, available soon ?

**Comment by hayaleturk** [comment]
@filfat Hey when does the snapchat application?

**Comment by gncgabriel** [comment]
@filfat  I have some doubts, the app will be in the Windows Store; if not, how people will do to install the app on their devices? because the app format is .APPXRECIPE and the only way to install it through visual studio (it's not very feasible), not fear tools such as "Windows Phone Deployer" (it does not accept .appxrecipe)

**Comment by kikothiago** [comment]
Windows in an already have the option to install straight from the appliance?

**Comment by gncgabriel** [comment]
@kikothiago  I could not do this

**Comment by azhang66** [comment]
Once we get #7 fixed you should be able to just download a .appx file and sideload it (hopefully)

**Comment by jimmyjango** [comment]
@gncgabriel - you can export the app as an appx and use Windows 10 SDK to deploy to the phone.

**Comment by Thomasfda** [comment]
The application not yet work

**Comment by jimmyjango** [comment]
That's my question to @filfat also. I've deployed on my W10 PC, can't seem to watch any videos. Also deployed to my W10 Phone and same result. Have you got playback working yet?

**Comment by jimmyjango** [comment]
Running on the desktop right now.

![snapchat](https://cloud.githubusercontent.com/assets/16617931/12224644/75e6ec2e-b7ee-11e5-8b07-428eb5786dbc.png)

**Comment by Thomasfda** [comment]
On my phone can not connect ..

**Comment by christurek** [comment]
In layman's terms, can someone explain to me the step by step process for how to use this app on my phone? I keep seeing compiling with visual studio to an .appx file then side loading but I don't know exactly what to do to do all of that. I'm more than willing to help test it on my new 950 (plus I just really want to be able to use snapchat) but I don't know how.

**Comment by pvrs12** [comment]
Right now, there is essentially nothing to test. You can login and view what snaps you have received (you can't see the actual snap, just that it exists). Right now the process for deploying it is a little convoluted and having developer experience helps a lot. Once the CI stuff starts working it should ease the process.

**Comment by jimmyjango** [comment]
As @pvrs12 said, there's nothing to test at all. It doesn't work yet.

**Comment by Thomasfda** [comment]
Connexion work ?

**Comment by hypertime123** [comment]
@Thomasfda the connection has always been working, its probably  a problem on ur end.

**Comment by Thomasfda** [comment]
it works ! It is not yet possible to see these snap will be keenly as possible!

**Comment by azhang66** [comment]
The CI now works [here](https://ci.appveyor.com/project/filfat/swiftsnapper) and you can now get .appx files at [here](https://ci.appveyor.com/project/filfat/swiftsnapper/build/artifacts).

**Comment by christurek** [comment]
I tested it. On my Lumia 950 it allowed me to log in and see who had snapped me. I couldn't take a picture and half of the camera screen was showing noise. it is, however, somewhat functional and it looks great. I'm really happy this is developing.

**Comment by Thomasfda** [comment]
Watch a snap does not work?

**Comment by Riegusto** [comment]
@Thomasfda it is stil an alpha build, so it is normal that lots of things work as they should. Pleas first read the comments before asking thing that are already explained.

**Comment by Thomasfda** [comment]
@Riegusto i wanted to be sure (sorry for my english)

**Comment by christurek** [comment]
![wp_ss_20160111_0002](https://cloud.githubusercontent.com/assets/16639378/12230928/8079988e-b808-11e5-874d-c1966d5e9c92.png)
![wp_ss_20160111_0004](https://cloud.githubusercontent.com/assets/16639378/12230930/80914b96-b808-11e5-9dd6-4a51f19e71c8.png)
![wp_ss_20160111_0008](https://cloud.githubusercontent.com/assets/16639378/12230929/808f9684-b808-11e5-8ec4-c5f148bb9c5c.png)
The splash screen shows a big X then changes to the swiftsnapper logo. The camera only takes up part of the screen and the app camera goes to blank/noise when then app is used after going to the action center or multitasking. Then if you try and switch the camera it crashes the app.

**Comment by MuriloPereira2015** [comment]
@christurek after you try switch the camera and the app crashes, can you open the app again?

**Comment by Riegusto** [comment]
You can open the app again, but after logging in you get 2 sec of noisy screen and than crashes again. Anyway that is what I Experience.

**Comment by MuriloPereira2015** [comment]
@Riegusto Yes, I have the same problem, after try switch the camera the app crashes and after open app again and do logon, it crashes again.

Device: Lumia 535 Windows 10 Mobile

**Comment by rphlcristian** [comment]
I can log however I can not take a photo and do not see the chat .
Device: Lumia 640 xl Windows 10 Mobile
(Sorry for my Inglês)

**Comment by filfat** [comment]
@rphlcristian We have yet to implement those features :)

**Comment by rphlcristian** [comment]
Ok :)

**Comment by sartero** [comment]
![hfdhdfh](https://cloud.githubusercontent.com/assets/16614542/12238301/798caf4a-b86a-11e5-845f-78bbe5617987.png)

What can it be ? Installed to build the application does not leave this screen.   :/

(Sorry for my Inglês)

**Comment by Riegusto** [comment]
This problem occurs only in the last build. I think it has to do with the Cordova plugin.

**Comment by sartero** [comment]
@Riegusto Then the problem is in the compilation, and not for lack of any plugin or app on my phone?

**Comment by Riegusto** [comment]
Indeed:)

**Comment by sartero** [comment]
Thanks a lot for the help. Just one doubts how do I install it on the PC? For the x86 and x64 station in .appx and not know how to install them.

**Comment by Riegusto** [comment]
@filfat after build 0.1.14 I was unable to login because it only shows the blue screen with the Swiftsnapper logo and word. Is this normal? I don't know anything about coding so i have no clue what it can be. i test almost any build but still only blue screen and logo.

using Lumia 625 W10M

**Comment by sartero** [comment]
@filfat , @Riegusto . The same thing happens to me, I just download the latest build but nothing to get access, since when I open the application it is only in blue logo SWIFTSNAPPER and does not present me nothing, nothing to click to enter or register.

Lumia 640 XL

**Comment by helderomendes** [comment]
Also on my Lumia 640XL, only appears this blue screen when the app is open. It's new compilation's fault

**Comment by filfat** [comment]
Looking into the issue :)

**Comment by Thomasfda** [comment]
On my L640 is good no blue screen :) !

**Comment by sartero** [comment]
@filfat OK, I'm happy to help .Espero soon be able to test it.

**Comment by VinzCHAOS** [comment]
You can only log in or also sign up with the app??
Anyway good work man!

**Comment by Thomasfda** [comment]
The blue screen is also on Lumia 930 (french friend report)

**Comment by helderomendes** [comment]
@Thomasfda, are you with the new compilation too?

**Comment by Thomasfda** [comment]
@helderomendes Yes with the new compilation, and I add the  Lumia 950 XL (build 10586.29) for blue screen (french friend report)

**Comment by Dafinek** [comment]
hello, is it working for build 10586.64? :) lumia 930

**Comment by Thomasfda** [comment]
Hello @Dafinek There are a blue screen with Lumia 930 but the application is not finish

**Comment by Dafinek** [comment]
@Thomasfda  i know, its not finish app :-) but thx for reply :p

**Comment by sartero** [comment]
Strange, I have to build the W10M 10586.63, and the latest version of APP SwiftSnapper and am having trouble with the blue screen. Good though it is an annoying little problem I'm glad for the work of developers, this icnado very good. an APP that many needed to time. - Lumia 640 XL -

**Comment by VinzCHAOS** [comment]
Lumia 950 latest insider build, also stuck on blue logo screen

**Comment by helderomendes** [comment]
Lets wait, its almost certainly that the issue is caused by the new compilation of W10 mobile, but I'm sure that is less complicated than we think. (Brazilian tester) - Lumia 640 XL

**Comment by gncgabriel** [comment]
the app no oppen in Mobile, only pc

**Comment by sartero** [comment]
@gncgabriel . How did you install it on the PC? If you can could inform me because I know only install on your phone.

**Comment by kikothiago** [comment]
![conceito](https://cloud.githubusercontent.com/assets/9504997/12251388/c8321f72-b8b5-11e5-98a2-e335c513a0cc.jpg)
                                                                                                                                                              Friends reconsider the application design concept. I would be very cool to be the APP was more familiar. Grateful

**Comment by kikothiago** [comment]
Of course, with the W10 features.

**Comment by allan991** [comment]
Nice! 👆

**Comment by gncgabriel** [comment]
![c](https://cloud.githubusercontent.com/assets/16600779/12253702/c32feb4a-b8c8-11e5-9be2-db06c366f045.jpg)

ta crashes when logging in both the mobile and the pc

**Comment by gncgabriel** [comment]
flagged yellow where ta giving error

**Comment by MuriloPereira2015** [comment]
![screenshot_209](https://cloud.githubusercontent.com/assets/10907608/12254074/0ee53c0e-b8cc-11e5-9071-955d9723b594.png)
@gncgabriel I receiving this error when I try compile the app, can you help me to solve that? I googled, but I don't found nothing to help me :/

**Comment by Thomasfda** [comment]
A blue screen with last compilation, on my lumia 640 :)..

**Comment by christurek** [comment]
Just tried latest build. Still stuck on splash screen on Lumia 950

**Comment by surya467** [comment]
Latest build works on windows 10 lates preview build.
Can login, camera opens but landscape view.

**Comment by filfat** [comment]
p.s, the latest build crashes while logging in; must have gotten something wrong with the progress indicator.

**Comment by christurek** [comment]
Still crashing upon login

**Comment by filfat** [comment]
It works now :)

**Comment by helderomendes** [comment]
Works now with the latest build?

**Comment by christurek** [comment]
![wp_ss_20160111_0010](https://cloud.githubusercontent.com/assets/16639378/12260310/0b3401b8-b8d0-11e5-9c3e-272692f030dc.png)
![wp_ss_20160112_0002](https://cloud.githubusercontent.com/assets/16639378/12260311/0b345028-b8d0-11e5-8eae-ec1e1a24fa95.png)
![wp_ss_20160112_0001](https://cloud.githubusercontent.com/assets/16639378/12260312/0ba660a0-b8d0-11e5-8684-014a9b260aa1.png)


Logged in just fine. Login process was very smooth this time. I enjoyed it. The only buttons that work are the stories, my snaps and switch camera buttons. Flash doesnt activate flash, idk what the emoji does and of course pictures cant be seen. But now it logs in just fine! Yay!

**Comment by christurek** [comment]
Camera is still tilted 90°

**Comment by surya467** [comment]
if app exits, login details are forgotten.

**Comment by boqus** [comment]
Where I can download this app?

**Comment by Riegusto** [comment]
@filfat 
On my lumia 520 with only rear camera it works fine in meaning that it does not crash. 
 On my lumia 625 with bothe rear camera and front facing camera it crashes immediately after login.

**Comment by kikothiago** [comment]
@filfat Please consider my concept for swiftsnapper. It would be great to be the APP had a familiar look.

**Comment by MuriloPereira2015** [comment]
@filfat App still crashing when I try switch the camera.

Lumia 535 Windows 10 Mobile

**Comment by filfat** [comment]
@kikothiago Sure, it would be nice to have a familiar look, then again creating our identity is ideal and dodging the copyright infringement bullet as much as possible.

Of course since this is an open source project we could add more options later, but that's not the focus now :)

**Comment by kikothiago** [comment]
I fully understand friend, was to get a little more beautiful. Forgiveness so;)

**Comment by christurek** [comment]
Swiping up on the stories screen sometimes moves you to the camera screen as if you swiped right to do so

**Comment by juniorlacerda** [comment]
I can't wait to get out the stable version!
Consider the concept of @kikotiago, is incredible.

**Comment by Dafinek** [comment]
i am still stuck on blue screen . :|

**Comment by MuriloPereira2015** [comment]
I'm having the same problem as the @Dafinek  is having after latter appx.

(sorry for my english)

**Comment by djerro123** [comment]
I just want to leave this message to express my appreciation to the developers, keep up the good work.! I don't use Snapchat myself, but I know a lot of people are missing this app from Windows 10 mobile. This is so great guys! Let me know when you start taking donations, i'll be happy to help fund this project.:)

**Comment by christurek** [comment]
If you want to donate you're more than welcome to on the main page for this. Click swiftsnapper at the top and click donate via PayPal. I think the link only shows up on mobile. Idk how to see it on desktop.

**Comment by sartero** [comment]
I'm glad to not more is stuck on blue screen, but since the last build'm not getting even open the APP, click on it and it opens and closes question of seconds alone. This is good staff, good job everyone, the APP is getting show.

**Comment by DARKFiB3R** [comment]
Just a quick note to wish you the best of luck. And to say F**K Evan Spiegel.

Should probably say F**K his service too, but apparently people want to play.

So yeah, stick it to the man ;)

**Comment by Vandorann** [comment]
Hi, not sure if it's been mentioned yet but will the UI buttons (buttons that take you to snaps and stories and the photo taking button) move with the navbar? So when the navbar is up, the three buttons aren't half half hidden by it. Cheers, and keep up the amazing work. :)

**Comment by Ahuge** [comment]
By nav bar do you meant he action center?

Or the soft buttons on the bottom for those with onscreen buttons?

**Comment by Vandorann** [comment]
@Ahuge the bottom buttons; back, home and search. That's the navbar :)

**Comment by Ahuge** [comment]
cool, I wasnt sure of the terminology.

I think we plan on accounting for it. Eith that or maybe collapsing them by default.

Thoughts?

**Comment by Vandorann** [comment]
@Aguge Oh awesome! Collapsing them by default would be cool, and could be a nice and easy 'fix' temporarily. But at the end of the day, adjusting the UI along with the navbar is much cleaner and elegant. Of course this is a very minor detail and should be pretty low priority, just thought I'd bring it up. :)

**Comment by Ahuge** [comment]
perfect, I think what I will aim for is hidden by default but when they come up, resize the gui with them

**Comment by YBTD** [comment]
I will try to do some testing in the next days, I'm also setting up my Visual Studio again, so maybe if I find the time, I can help a little.

**Comment by realfakenerd** [comment]
Guys, will you guys a in app update? Just to don't need to download and sideload on every single update

**Comment by VinzCHAOS** [comment]
Where i can download the latest appx to test it on my lumia 950?

**Comment by mentos1386** [comment]
@VinzCHAOS download one that has "arm" in the name. https://ci.appveyor.com/project/filfat/swiftsnapper/build/artifacts

**Comment by MuriloPereira2015** [comment]
Some information about last build into Lumia 535 - Windows 10 Mobile

- After I installed last build uploaded to AppVeyor, the app don't crashes when I try switch the camera, but, the camera freezes the image and doesn't change.

- After I try change the camera and app freezes the image, when I restart the app, camera doesn't work more.

- Camera doesn't fill full screen.

**Comment by sartero** [comment]
@Pereira2015  .. Funny on my Lumia 640 XL - Windows 10 Mobile - not occore none of these errors cited by you friend, here the camera is running smoothly, fills the entire screen and do not lock the switch to front or rear.

Sorry for my English.

**Comment by realfakenerd** [comment]
It's happening to me the same thing on L535

**Comment by Riegusto** [comment]
@realfakenerd try to restart your phone, this might help. worked for me

**Comment by helderomendes** [comment]
@realfakenerd Dou you have the latest build?

**Comment by realfakenerd** [comment]
Yep, the latest version

**Comment by realfakenerd** [comment]
Guys, another bug, I login and then when I close the app, then I open it again and isn't logged in, so I have to login every time I want to use the app.

**Comment by azhang66** [comment]
@realfakenerd I'm working on that. Give it some time

**Comment by M0ti0n** [comment]
So this currently only works for windows 10? Will there be version for wp 8.1 later on?

**Comment by allan991** [comment]
Discussion totally [OFF] Windows 10 is release by T-Mobile in Poland. Its official.

**Comment by realfakenerd** [comment]
Azhang, so I'm waiting

**Comment by realfakenerd** [comment]
Motion, on windows 10 is way better to install the app, I don't even need a PC to install on my Lumia, and dude is Windows 10, the latest version, so I think everybody will have the Windows 10

**Comment by M0ti0n** [comment]
yeah adopting it for silverlight in wp 8.1 would be PITA I guess. We shall se how fast the adoption of bug infested w10 will be.

**Comment by Thomasfda** [comment]
Read the snap coming soon? :D

**Comment by gncgabriel** [comment]
@filfat  the app is crashing on the blue screen (appears before the login), mobile test

**Comment by filfat** [comment]
![](http://i.imgur.com/rDBVybC.png) :D

**Comment by Riegusto** [comment]
@filfat woop woop filfat for the win!! eager to test this :D :+1:

**Comment by gncgabriel** [comment]
:+1:  :DDDDDDDDDDDDDD

**Comment by allan991** [comment]
👏👏👏👏

**Comment by Riegusto** [comment]
@filfat I was able to open a snap but than the application froze, any way the adrenaline rush was huge!!!

**Comment by realfakenerd** [comment]
Still having the some annoying bugs, but dude, you're awesome

**Comment by Thomasfda** [comment]
Work for me to my L1320 but the photo remaining indefinitely

**Comment by TrueBlueAura** [comment]
The latest build worked on my Lumia 1520.

**Comment by Mrmarley11** [comment]
Hey. Downloaded the latest build and clicked install on my 950XL but can't find the app? What am I doing wrong?  

And thought about if you could make a to-do-list (Can't find one?) where you have described what has been implemented, what you're working on now and what features will you implement in the future? It could be easier to catch up on the whole process!

But I'm appreciating everyone's work! (Y) Keep it up!

**Comment by tolap95000** [comment]
i Download the APPX of @mentos1386 , only CordovaApp.Windows10_1.0.0.0_arm_debug works to install.

But when i launch, i put name and password, and go, then the application exit.

I'm on Lumia 1520 Windows 10 1.3.231.4

**Comment by tolap95000** [comment]
@TrueBlueAura , version do you have ?

**Comment by VinzCHAOS** [comment]
When are u going to ability sign in option??

**Comment by tolap95000** [comment]
when ?

i click on application, and i click on "connexion" (french) then i can sign in.

**Comment by Thomasfda** [comment]
Yes.. Hum i think i's a api bug.
(un bug de l'api quoi)

**Comment by gherb** [comment]
@tolap95000 I've got the same issue. I sign in and then the app exits. 
Great work anyway!

**Comment by helderomendes** [comment]
It's normal this, on the last build, @filfat made changes to prepare for the new casper .api, so the app is temporarily unfunctional (I think it's because of this)

**Comment by TrueBlueAura** [comment]
@tolap95000 @gherb
For me, the app would exit on sign-in if my login information was incorrect.

**Comment by TrueBlueAura** [comment]
@tolap95000 I was using the current ARM build from, https://ci.appveyor.com/project/filfat/swiftsnapper/build/artifacts

**Comment by gherb** [comment]
@tolap95000 It doesnt work for me. The app just exits. No password or username mistakes

**Comment by gherb** [comment]
Should be @TrueBlueAura

**Comment by filfat** [comment]
API.Casper.Io is currently down.

**Comment by sartero** [comment]
@filfat  . The APP this with a small error in the log, where when I click enter it closes the APP of nowhere, already tried several times and nothing to enter, the application only closes. But the project is digging otimo, congratulations to all q are helping. - Lumia 640 XL, with the latest build of SwiftSnapper -

**Comment by VinzCHAOS** [comment]
I managed to log in with my 950, but the camera doesn't fit full resolution of screen, and i still can't add or find friend, if i press the smile icon it don't do anything, but at least i can take snap, sorry for my bad english anyway, keep the good work

**Comment by sartero** [comment]
@VinzCHAOS  . He was able to log in? What version of APP is using?

**Comment by VinzCHAOS** [comment]
@sartero i think is the latest, i downloaded it yesterday night, yes, i sign up from the phone of a friend because i didn't have an account then try to log in and it worked!

**Comment by TrueBlueAura** [comment]
On my Lumia 1520, I was able to: 
- sign-in.
- get the "Picture successfully taken" message.
- see that Team Snapchat sent me a snap, but the picture wouldnt load. I assume it's just a video that the app isn't compatible with yet.

**Comment by tolap95000** [comment]
@TrueBlueAura, what is your WP version ?

**Comment by Riegusto** [comment]
this is a video of a working swiftsnapper build in action :D (build 0.1.141)

https://www.youtube.com/watch?v=PHtS8WDqPEY

**Comment by TrueBlueAura** [comment]
Windows 10 Mobile
Version 1511
OS Build 10.0.10586.63

**Comment by allan991** [comment]
Yeah! Front camera its working here but the quality's it's low I don't know why that's because Win10 still preview or something else

**Comment by allan991** [comment]
I gonna work with solutions now with take a videos and Stories

**Comment by kikothiago** [comment]
O projeto acabou? :(

**Comment by trischmateus** [comment]
@kikothiago nao, só q API usada foi atualizada e o projeto ta parado por um tempo ate acharmos como terminar de programar ele

**Comment by christvedt** [comment]
Where can I download .appx for build 0.1.141?

**Comment by christvedt** [comment]
Never mind. Found it. But cannot receive or send yet. It's looking great though! Cameras working great, both front and rear on my Lumia 930.

**Comment by christvedt** [comment]
Edit: Not sure if my login is correct. I get the same experience whatever I try to log in as. Seems incorrect login is not detected.

**Comment by tolap95000** [comment]
Don't work for mee :'(

**Comment by allan991** [comment]
Hey @filfat whats going on with the Project ?

**Comment by Riegusto** [comment]
@allan991 Dude give the man a break will ya. He has a live and other projects beside swiftsnapper. This message is to all people here, stop complaining and asking why it is not working any more or why this and why that if you can find all your answers in the issue section or even in this discussion. Unles you find a real bug or something that had been changed without any report about it, than you may ask something...

Ps have a good weekend to you all. ;)

**Comment by filfat** [comment]
@allan991 Between working on http://www.playstr.link & singing I'm researching the token generation. The API updates on this project will be quite stale till Casper's API goes public or till we find another way into Snapchat's API.

**Comment by allan991** [comment]
Really nice! @filfat tell me more about playstr, but not here of curse if you have twitter fallow me @allan_991 then tell me more :)

**Comment by helderomendes** [comment]
Any chance to call uncle Rudy to help us? Maybe he has a solution. I'm asking because maybe someone here has a little influence over him or maintain contact

**Comment by christurek** [comment]
Rudy doesn't give a shit about anything Snapchat related anymore. He had lawyers at his house because of it

**Comment by filfat** [comment]
Hi, sorry for the extreme inactivity lately.
I'll do my best to work on this project on weekends from her on out, however recently I got indeed a few very important work-related projects that goes before a OSS hobby project (One of them is Playstr, the other one I can't discuss as of now, but it's not a fun project).

On the API end, we'll see how we solve this issue going forward.
Also, from here on out I would recommend everybody to use the subreddit over at https://www.reddit.com/r/swiftsnapper instead of this issue (as it takes an increasingly long time to load this page even on my beefy workstation).

Cheers :)
